fdk/fixtures.py

```diff
@@ -12,36 +12,43 @@
 #    License for the specific language governing permissions and limitations
 #    under the License.
 
-import asyncio
 import datetime as dt
 
 from fdk import constants
 from fdk import headers as hs
-from fdk import http_stream
+from fdk import runner
 
 
 async def process_response(fn_call_coro):
     new_headers = {}
     resp = await fn_call_coro
-    for k, v in dict(resp.headers).items():
+    response_data = resp.body()
+    response_status = resp.status()
+    response_headers = resp.context().GetResponseHeaders()
+    for k, v in response_headers.items():
         new_headers.update({
             k.lstrip(constants.FN_HTTP_PREFIX): v
         })
-    content = await resp.content.read()
-    resp_headers = hs.decap_headers(resp.headers)
-    status = int(resp.headers.get(constants.FN_HTTP_STATUS))
-    resp_headers.delete(constants.FN_HTTP_STATUS)
-    return content, status, resp_headers
+    print(response_headers)
+    resp_headers = hs.decap_headers(response_headers)
+    del resp_headers[constants.FN_HTTP_STATUS]
```

Comment:
We could get a `KeyError` here if we manage to get `resp_headers` without a `FN_HTTP_STATUS` field.  Should this code attempt to be resilient to this kind of failure?  Or at least give some kind of actionable error message?